webapp/utils/constants.jsx

```diff
@@ -849,7 +849,8 @@ export const Constants = {
     MENTION_MEMBERS: 'mention.members',
     MENTION_NONMEMBERS: 'mention.nonmembers',
     MENTION_SPECIAL: 'mention.special',
-    DEFAULT_NOTIFICATION_DURATION: 5000
+    DEFAULT_NOTIFICATION_DURATION: 5000,
+    STATUS_INTERVAL: 15000
```

Comment:
Why 15 sec,  why not 30 or 1 minutes.

20K users will be calling this every 15 seconds in the current tests?